tests/e2e/pageObjects/AccountNameEditPage.js

```diff
@@ -0,0 +1,42 @@
+const { format } = require("util");
+
+const config = require("../../cucumber.conf.js");
+
+class AccountNameEditPage {
+
+  constructor() {
+    this.baseUrl = config.tandoorURL;
+    this.settingURL = this.baseUrl + 'search/'
+    this.spaceOverviewURL = this.baseUrl + 'settings/';
+    this.navBarMenuSelector = '//i[contains(@class,"fa-user-alt")]//ancestor::a';
+    this.settingLinkSelector = '//a[@href="/settings/"]';
+    this.accountLinkSelector = '//a[text()=" Account "]';
+    this.firstNameSelector = '#__BVID__37';
+    this.lastNameSelector = '#__BVID__39';
+    this.succesMsgSelector = '//div[@class="toast-body"]';
+    this.navBarNameSelector='//a[contains(text()," %s ")]';
+  }
+
+  async goToSettingPage() {
+    await page.click(this.navBarMenuSelector);
+    await page.click(this.settingLinkSelector);
+    await page.click(this.accountLinkSelector);
+  }
+
+  async fillUserData(firstname, lastname) {
+    await page.fill(this.firstNameSelector, firstname);
+    await page.fill(this.lastNameSelector, lastname);
+    await page.locator(this.firstNameSelector).focus();
+  }
+
+  async getUserName() {
+    await this.goToSettingPage()
+    return await page.locator(this.navBarMenuSelector).innerText();
+  }
+
+   getSuccessMsg() {
+    return  page.locator(this.succesMsgSelector).innerText();
+  }
```

Comment:
formatter could save save a lot of time
if using vscode, use this settings:
```json
{
...
  "editor.formatOnSave": true,
...
}
```

```suggestion
   getSuccessMsg() {
    return page.locator(this.succesMsgSelector).innerText();
  }
```